main/ui/src/main/java/org/cryptomator/ui/fxapp/UpdateCheckerModule.java

```diff
@@ -69,6 +73,7 @@ static HttpRequest provideCheckForUpdatesRequest(@Named("applicationVersion") Op
 				return new UpdateCheckerTask(httpClient, checkForUpdatesRequest);
 			}
 		};
+		service.setOnFailed(event -> LOG.error("Failed to execute update service", service.getException()));
```

Comment:
Imho this line and adding a logger is not necessary, since the default fail action for UpdateChecker task is logging the error, or am I wrong?

Reply:
It is not necessary, but it adds additional safety in case the task is reused without adding a new listener.